transit.md

```diff
@@ -48,7 +48,8 @@ This may be relaxed in the future, much as Wormhole was.
 
 Each Transit object has a set of "abilities". These are outbound connection
 mechanisms that the client is capable of using. The basic CLI tool (running
-on a normal computer) has two abilities: `direct-tcp-v1` and `relay-v1`.
+on a normal computer) has these abilities: `direct-tcp-v1`, `relay-v1`,
+`tor-tcp-v1`, `direct-ws-v1` and `direct-wss-v1`.
```

Comment:
I know it's kind of tangential to your change, but why would one advertise any of the other three values? What meaning do they have in the protocol/handshake? I think the only relevant question is "can we connect directly or do we need a relay"?

Reply:
What you are saying makes sense, now that relay specification also includes the protocol. It wasn't there in the past before the WebSocket changes.

Reply:
Perhaps those changes should be separate from this change as a "-v2" abilities? That would also help maintain backward compatibility with older relay servers and clients.